ebean-test/src/test/resources/migrationtest/dbmigration/cockroach/1.1.sql

```diff
@@ -7,6 +7,9 @@ alter table migtest_e_basic drop constraint if exists ck_migtest_e_basic_status2
 drop index uq_migtest_e_basic_indextest2 cascade;
 drop index uq_migtest_e_basic_indextest6 cascade;
 alter table migtest_e_enum drop constraint if exists ck_migtest_e_enum_test_status;
+alter table if exists drop_main_drop_ref_many drop constraint if exists fk_drop_main_drop_ref_many_drop_main;
```

Comment:
If this is not here, `ebean-test/src/test/resources/migrationtest/dbmigration/cockroach/1.2__dropsFor_1.1.sql` would fail